command_test.go

```diff
@@ -148,6 +148,38 @@ func TestSubcommandExecuteC(t *testing.T) {
 	}
 }
 
+func TestSetContext(t *testing.T) {
```

Comment:
Thanks for adding a test case for this!! 👍

Reply:
:+1: